requirements.txt

```diff
@@ -1,3 +1,55 @@
 Django==2.1
 feedparser==5.2.1
 pytz==2018.5
+## The following requirements were added by pip freeze:
```

Comment:
wait where did these come from?

Reply:
When I run pip freeze I only get the three requirements above. Are you in a virtualenv? Maybe these are system wide pip installs?

Reply:
oh christ. these are from the ubuntu/bionic64 base image.
didn't think the system packages would be so polluted, my b

Reply:
Haha all good, will merge once adjusted